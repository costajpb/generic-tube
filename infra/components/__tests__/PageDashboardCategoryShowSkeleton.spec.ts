import snapshowWrapper from "@/infra/__tests__/snapshotWrapper"
import PageDashboardCategoryShowSkeleton from "../PageDashboardCategoryShowSkeleton.vue"

describe('infra/components/PageDashboardCategoryShowSkeleton', () => {
    it('should render', () => {
        expect(snapshowWrapper(PageDashboardCategoryShowSkeleton, {}, {
            name: 'Category',
            shows: []
        }, {
            useCase: {
                display: () => {}
            }
        })).toMatchSnapshot()
    })
})