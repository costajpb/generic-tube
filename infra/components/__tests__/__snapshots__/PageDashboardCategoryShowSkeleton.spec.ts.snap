// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/PageDashboardCategoryShowSkeleton > should render 1`] = `
<li
  class="_wrapper_587f33 _skeleton_bdd73e"
  name="Category"
  shows=""
>
  
  
</li>
`;
